### PR TITLE
fix(NewCollectiveModal): Mention that own user is part of collective

### PR DESCRIPTION
**Pull request by mejo-**
People got confused by searching their own user.

#### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
screenshot | recording1

### 🏁 Checklist

- [x] Code is properly formatted (`npm run lint` / `npm run stylelint` / `composer run cs:check`)
- [x] [Sign-off message](https://probot.github.io/apps/dco/) is added to all commits

### CODE REVIEW

File: src/components/Nav/NewCollectiveModal.vue [hunk 1]
```diff
@@ -87,6 +87,7 @@
 					<MemberPicker :show-selection="true"
 						:selected-members="selectedMembers"
 						:on-click-searched="onClickSearched"
+						:empty-content-hint="t('collectives', 'Your user is added automatically.')"
```
Comment (mejo-) [inline src/components/Nav/NewCollectiveModal.vue:90]: Not sure about the wording. Maybe "As creator, you're already added." is better? :thinking:

**Review by jancborchardt — COMMENTED** [review]
Instead of as a message, we could show the owner's account directly below in the list like added accounts do? That would make it more obvious than a line of text.

Slightly related: We want to move from the wording "user" to either "account" or "person", could be replaced in the search field.

**Comment by mejo-** [comment]
Thanks @jancborchardt, much better. Implemented now, see the screencast above. I also changed "users" to "accounts" where appropriate.

**Comment by mejo-** [comment]
I wonder whether it would further help to change the "Create" button to something like "Create without adding people" in case no other members were selected.

**Comment by juliushaertl** [comment]
Small addition, we could preserve the previously entered collective name in the dialog title to make it more obvious what this is about, e.g. "Add people to fox" instead of just "Add people".

Overall i find the different usages of account/people/members a bit confusing. 

Screenshot 2023-11-09 at 19 56 28


Screenshot 2023-11-09 at 19 55 35

Screenshot 2023-11-09 at 19 55 54

Maybe we can somehow unify that? Pick one high level word (members) and a fixed set for the actual membership entities (account, group, circle)?

**Comment by mejo-** [comment]
> Overall i find the different usages of account/people/members a bit confusing.
> Maybe we can somehow unify that? Pick one high level word (members) and a fixed set for the actual membership entities (account, group, circle)?


I agree, thanks for bringing this up. Your suggestion makes sense: account, group, circle for the different entities and "members" for all of them. So I'd replace "Add people" with "Add members" and search for more occurrences of "users" to be replaced by "accounts". Agreed?

**Comment by mejo-** [comment]
> Small addition, we could preserve the previously entered collective name in the dialog title to make it more obvious what this is about, e.g. "Add people to fox" instead of just "Add people".

Great idea. Implemented now (using "Add members to <name>"). See updated screencast above.

> I agree, thanks for bringing this up. Your suggestion makes sense: account, group, circle for the different entities and "members" for all of them. So I'd replace "Add people" with "Add members" and search for more occurrences of "users" to be replaced by "accounts". Agreed?

Also implemented. See updated screencast above.

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**